labtool2.0/src/components/LabtoolComment.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { Button, Icon, Comment, Label } from 'semantic-ui-react'
+import ReactMarkdown from 'react-markdown'
+import { withRouter } from 'react-router'
+import { connect } from 'react-redux'
+import { trimDate } from '../util/format'
+
+export class LabtoolComment extends React.Component {
```

Comment:
This could be functional component

Reply:
it is better to have these as class components for now with a functional component conversion being done in a later refactoring commit, converting _all_ components into being functional.